Disposition error integrated into cleaning folder

jdunca opened the pull request:
Hey Lucas, 

Follow up on my message from yesterday. I integrated the disposition error cleaning function into the cleaning folder (you have a really janky version, whcih I have now cleaned up).

Cheers,
Jamie

Commit:
disp error concept

Commit:
Create .DS_Store

Commit:
integrated disp_error into cleaning

lchski commented:
Thanks @jdunca – it looks like the changes maybe didn’t survive the commits ([see the “Files changed” tab for what I mean](https://github.com/lchski/gc-ati-summaries-data/pull/8/files); it’s mostly `.DS_Store` files). I tried to look at the first commit, but I don’t totally understand how it’s cleaning the dispositions.

Commit:
Add ati_summaries-jd.csv

Found broken entries in old summaries file (several DOJ entries from 2018). Moved them to correct columns but they are still missing ATIP number.

Commit:
Update ati-summaries.csv

jdunca commented:
Sorry about that Lucas, I have no idea what I was thinking at the time. I've found a way (I think) to resolve several of the issues that trace back to some mis-placed DOJ entries from back in 2018. I've manually just moved them in the source file and re-merged the .csv of old requests. I then worked this into the ati-summaries.csv in my branch.
